### DISCUSSION TITLE
Big Problem !

### DISCUSSION THREAD

**Discussion by ShadowOffDeath** [rejetto/hfs#574] [General]
Massiomo I don`t know why byt version 0.52.2 and 0.52.3 my Kaspersky Antivirus is Deleted like
![HFS](https://github.com/rejetto/hfs/assets/168378955/3840d0ed-abbb-4766-a06b-bee366b47c66)
Can you fix that problem Please!

**Comment by PH5HP** [comment]
Hi,

Can't you make an exception for HFS in Kaspersky Antivirus?

Kind regards,
Henk - ph5hp..nl - fileserver.ph5hp.nl

**Reply by ShadowOffDeath** [reply]
I try but if i update online HFS  program Cannot Stop Antivirus to Delete New Exe of HFS. Maby you or i need to report to Kaspersky for False Detection ? You tell me i or You Send Report to Kaspersky?

**Reply by PH5HP** [reply]
Hi,

You can always try to file a report to flag HFS as safe by Kaspersky.

Kind regards,
Henk - ph5hp..nl - fileserver.ph5hp.nl

**Comment by rejetto** [comment]
have a look at https://github.com/rejetto/hfs/wiki/Antivirus

i don't know how to solve this problem, but basically it's a problem of kaspersky, not of HFS. It's them reporting something false.

**Comment by ShadowOffDeath** [comment]
Hi again i cannot send file hfs to kaspersky for report false detection. Can any other user tru to report him please!
New 0.52.4 Aghain Detect Trojan.Win32.Pits.bdg !!!
Why with version 0.52.0 Work Fine and Not Virus Detections but All Others New Version Detect Trojan.Win32.Pits.???
Maby you made some wrong Please check what you changes 0.52.0 to 0.52.4 or i think or some hacker mod you files and put to download here githubs not safe files plase!!!

**Reply by PH5HP** [reply]
Hi,

My virus scanner does not detect a virus in HFS. So I still think Kasperky flags HFS wrongly containing a virus.

Kind regards,
Henk - ph5hp..nl - fileserver.ph5hp.nl

**Comment by rejetto** [comment]
i cannot answer "why", you should ask them, but i guess they consider 0.52.0 fine because some people have been reporting it.
this is always happening, not just with 0.52, that's why I'm sure i did nothing wrong this time.

**Comment by ShadowOffDeath** [comment]
Bravo Massimo!
Ho postato una domanda sul loro forum https://forum.kaspersky.com/forum/kaspersky-free-73/?
Non ci resta che aspettare che uno dei loro dipendenti legga e reagisca a quanto scritto e smetta di cancellare il tuo bel programma che sviluppi da molti anni con tanto amore e attenzione, di cui ti sono molto grato.
Good Massimo!
I posted a query on their forum https://forum.kaspersky.com/forum/kaspersky-free-73/?
We just have to wait for one of their employees to read and react to what was written and to stop deleting your nice program that you have been developing for many years with a lot of love and attention, for which I am very grateful.

**Comment by rejetto** [comment]
my suggestion is to exclude the whole folder, so that you are not annoyed with future updates.

I've heard that HFS (version 2 at least) has been used as a tool in some malicious activities but, as any tool, it can be used for good or evil. Unaware if it was installed by the user or an attacker, some antiviruses decide to just report or even delete it.

**Reply by ShadowOffDeath** [reply]
Sono tante nuove novità! Come scanner online del file 0.52.4 nel sito tecnico
https://opentip.kaspersky.com/640B19AAFE482438B610A00C039B3463B2B6AE04B753EBCD8A23DF9C349F14A4/results?tab=upload
Il primo file è protetto. La versione online standard e lo scanner hanno un antivirus che non si apre e non si blocca.
Tieni presente che i russi non possono eseguire rilevamenti falsi sul tuo programma con i programmi antivirus.
Ma la mia risposta al forum tecnico è stata modificata e c'è stato un problema di adattamento a tutti i tipi di utilizzo tutti gli hfs.
Let's see what happened! Read more on page 0.52.4 on this page
https://opentip.kaspersky.com/640B19AAFE482438B610A00C039B3463B2B6AE04B753EBCD8A23DF9C349F14A4/results?tab=upload
Let's see what happened. Let me tell you what the story is I'll let you know and I'll let you know.
Let me tell you what you have done before this is what you want to know about.
Let me tell you what you have done and what you will do. Read more about the hfs.

If some wont to check files before instal you can chek online in https://opentip.kaspersky.com/

**Comment by rejetto** [comment]
i guess the problem will happen again with 0.52.5, etc etc 🫠

**Reply by ShadowOffDeath** [reply]
Massimo, spero che questo problema non si verifichi più, anche se non ho una risposta ufficiale dal loro forum. Tuttavia, c'è un po' di supporto e modi per contattarli. Dovrei chiudere questa discussione o lasciarla aperta? Cosa ne pensi?
Massimo, I hope this problem doesn't happen again, even though I don't have an official response from their forum. However, there is some support and ways to contact them. Should I close this discussion or leave it open? What do you think?

**Reply by rejetto** [reply]
i never close "discussions", just the "issues"

**Comment by ShadowOffDeath** [comment]
I Care to report  new version of HFS file server  and cannot detect virus again!

**Reply by rejetto** [reply]
that sounds good 🤞
